package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hapi-auth-ip-whitelist",
-  "version": "0.0.5",
+  "version": "0.0.6",
```

Comment:
Don't update the version, we do this when releasing.

Reply:
Ok, noted.